lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1160,26 +1160,27 @@ def set_http_request_grpc(record, entry)
       input = record['httpRequest']
       output = Google::Logging::Type::HttpRequest.new
       output.request_method = input.delete('requestMethod') if
-        input.key?('requestMethod')
+        input.key?('requestMethod') && !input['requestMethod'].nil?
```

Comment:
Without extra parameters on construction, the default return value of a Hash lookup is `nil`. So this is equivalent to: `if !input['request_method'].nil?` (i.e., the `key?` call is redundant). Also everywhere below.